code/datums/diseases/advance/advance.dm

```diff
@@ -294,6 +294,8 @@
 		var/list/L = list()
 		for(var/datum/symptom/S in symptoms)
 			L += S.id
+			if(S.neutered)
+				L += "N"

```

Comment:
why is this here, the change was already merged

goddamn git and goddamn branches of branches